webapp/src/app/course-welcome/course-welcome.component.ts

```
@@ -140,6 +201,200 @@
+      // console.log('Resizing and uploading file with the name: ' + newFileName);
+
+      const image = new Image();
+      const reader = new FileReader();
+
+      // Read the chosen file
+      reader.onload = (e: any) => {
+        image.src = e.target.result; // Set the image source to the file data
+        image.onload = () => {
+          // Create a canvas to resize the image
+          const canvas = document.createElement('canvas');
+          const ctx = canvas.getContext('2d')!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)